cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,38 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
 
-  });
+const ArticlePage = new ArticlePageObject();
+describe('Article', () => {
+  let user;
+  let article;
 
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user);
+      cy.task('generateArticle').then((data) => {
+        article = data;
+        cy.visit('/#/editor');
+      });
+    });
   });
 
   it('should be created using New Article form', () => {
-
+    ArticlePage.createArticle(article);
   });
 
   it('should be edited using Edit button', () => {
-
+    ArticlePage.createArticle(article);
+    cy.task('generateArticle').then((newArticle) => {
+      ArticlePage.editArticle(article.title, newArticle);
+    });
   });
 
   it('should be deleted using Delete button', () => {
-
+    ArticlePage.createArticle(article);
+    ArticlePage.deleteArticleButton.click();
+    cy.contains('Deleted the article. Going home...');
```

Comment:
Ensure that the text 'Deleted the article. Going home...' is exactly what appears on the page after deletion. If the actual message differs, the test will fail. Consider verifying the exact message or using a partial match if the message might change.